### PR TITLE
pls approve

### PR DESCRIPTION
**Pull request by RAYXSTUDIOZGAMES**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/RAYXSTUDIOZ-DEV.json
Comment (CuteDog5695) [inline domains/RAYXSTUDIOZ-DEV.json:1]: .json file should be lowercase

File: domains/dungeonsbyrayan
Comment (CuteDog5695) [inline domains/dungeonsbyrayan:1]: file isnt .json

File: domains/sussyrayan.json
Comment (CuteDog5695) [inline domains/sussyrayan.json:1]: invalid file

File: sussyrayan.json
Comment (CuteDog5695) [inline sussyrayan.json:1]: this is a invalid .json file

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/dungeonsbyrayan
Comment (CuteDog5695) [reply domains/dungeonsbyrayan:1]: oh also you cant add files outside of domains folder

File: sussyrayan.json
Comment (CuteDog5695) [reply sussyrayan.json:1]: oh also you cant add files outside of domains folder

**Pull request closed without merging**